react-file-uploads/server/server.js

```diff
@@ -0,0 +1,65 @@
+import 'dotenv/config';
+import express from 'express';
+import pg from 'pg';
+import cors from 'cors';
+import ClientError from './lib/client-error.js';
+import errorMiddleware from './lib/error-middleware.js';
+import uploadsMiddleware from './lib/uploads-middleware.js';
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+
+app.use(cors({ origin: '*' }));
+app.use(express.static('public'));
+app.use(express.json());
+
+app.post('/api/uploads', uploadsMiddleware.single('image'), async (req, res, next) => {
+  try {
+    const { caption } = req.body;
+    if (!caption) {
+      throw new ClientError(400, 'caption is a required field');
+    }
+    /* TODO:
+    * - create a url for the image by combining '/images' with req.file.filename
+    * - insert the "caption" and "url" into the "images" table
+    * - respond with the inserted row data and the proper status code for inserting data
+    */
+    const url = `/images${req.file.filename}`;
```

Comment:
The URL needs a slash after /images or it won't be able to find the file in the images directory.